components/EditForm.jsx

```diff
@@ -0,0 +1,33 @@
+import StyledForm from "./Styles/StyledForm";
+import StyledInput from "./Styles/StyleInput";
+import StyledDiv from "./Styles/StyledDiv";
+import PositionedButton from "./Styles/StyledButton";
+
+export default function EditForm({ selectedPet, handleMode, onSubmit }) {
+  return (
```

Comment:
if you adjust your code according to my previous comments, handleMode needs to replaced here as well